control-plane-operator/controllers/hostedcontrolplane/clusterpolicy/params.go

```diff
@@ -42,6 +42,7 @@ func NewClusterPolicyControllerParams(hcp *hyperv1.HostedControlPlane, globalCon
 	params.DeploymentConfig.SetColocation(hcp)
```

Comment:
I wonder if there's any valuable unit test we can add for all these?

Reply:
yes, these should be unit-testable, let me work on some simple unit tests